processor/configuration/src/main/resources/schemas/actions/google_pubsub_sink_0.1.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "gcp_project_id",
+    "gcp_destination_name",
+    "gcp_service_account_key"
+  ],
+  "properties": {
+    "gcp_project_id": {
+      "title": "Project Id",
+      "description": "The Google Cloud PubSub Project Id",
+      "type": "string"
+    },
+    "gcp_destination_name": {
+      "title": "Destination Name",
+      "description": "The Destination Name",
+      "type": "string"
+    },
+    "gcp_service_account_key": {
+      "title": "Service Account Key",
+      "description": "The Service account key that can be used as credentials for the PubSub publisher/subscriber",
+      "type": "string",
+      "x-group": "credentials",
+      "format": "base64"
```

Comment:
Thinking whether we should tell user to encode the service account key also in description, not just here in format....

Reply:
Added in the description